consensus/istanbul/backend/announce.go

```diff
@@ -203,11 +205,11 @@ func (sb *Backend) handleIstAnnounce(payload []byte) error {
 	}
 
 	// If the message is not within the registered validator set, then ignore it
-	regVals, err := sb.retrieveRegisteredValidators()
+	regVals, err := validators.RetrieveRegisteredValidators(nil, nil)
```

Comment:
Is it okay to not use pending state in announce?

Reply:
Ya, it's fine to use the latest finalized block on the blockchain.